### PR TITLE
Add config field for stats update interval.

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Add config field for stats update interval, so that we can control the frequency to update check, quota and report stats at proxy such as Envoy.

@qiwzhang

### CODE REVIEW

File: mixer/v1/config/client/client_config.proto [hunk 1]
```diff
@@ -70,6 +70,10 @@ message TransportConfig {
   }
   // Specifies the policy when failed to connect to Mixer server.
   NetworkFailPolicy network_fail_policy = 4;
+
+  // Specifies interval in milliseconds at which stats maintained by proxy for
+  // check, quota and report calls get updated.
```
Comment (qiwzhang) [inline mixer/v1/config/client/client_config.proto]: Please add
// if not specified, or 0 protobuf default value, the interval is 10 seconds

File: mixer/v1/config/client/client_config.proto [hunk 2]
```diff
@@ -70,6 +70,10 @@ message TransportConfig {
   }
   // Specifies the policy when failed to connect to Mixer server.
   NetworkFailPolicy network_fail_policy = 4;
+
+  // Specifies interval in milliseconds at which stats maintained by proxy for
```
Comment (qiwzhang) [inline mixer/v1/config/client/client_config.proto]: Specify fresh interval in milliseconds to write mixer client statistics to Envoy share memory.

**Comment by JimmyCYJ** [comment]
Please take a look. Thanks!

File: mixer/v1/config/client/client_config.proto [hunk 3]
```diff
@@ -70,6 +70,11 @@ message TransportConfig {
   }
   // Specifies the policy when failed to connect to Mixer server.
   NetworkFailPolicy network_fail_policy = 4;
+
+  // Specify fresh interval in milliseconds to write mixer client statistics to
```
Comment (ayj) [inline mixer/v1/config/client/client_config.proto]: s/fresh/refresh

**Pull request merged**